test-requirements.txt

```diff
@@ -1,9 +1,7 @@
-mock
-nose
 coverage
-pep8
-pylint >= 1.1.0
 ddt
 diff-cover >= 0.2.1
--e .
--r requirements.txt
+edx-lint>=0.4.0,<1.0.0
+mock
```

Comment:
Yeah ranges are fine by me.

On Wed, Jan 27, 2016 at 2:42 PM, Clinton Blackburn <notifications@github.com
> wrote:

> In test-requirements.txt
> <https://github.com/edx/ccx-keys/pull/1#discussion_r51038511>:
>
> >  ddt
> >  diff-cover >= 0.2.1
> > --e .
> > --r requirements.txt
> > +edx-lint>=0.4.0,<1.0.0
> > +mock
>
> Done. @benpatterson <https://github.com/benpatterson> thoughts on using
> ranges rather than exact versions for test requirements?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/edx/ccx-keys/pull/1/files#r51038511>.
>